app/presenters/formats/edition_format_presenter.rb

```diff
@@ -1,12 +1,13 @@
 module Formats
   class EditionFormatPresenter
+
     def initialize(edition)
       @edition = edition
       @artefact = edition.artefact
     end
 
-    def render_for_publishing_api(republish: false)
-      required_fields(republish).merge optional_fields
+    def render_for_publishing_api(republish: false, rendering_app: "frontend")
```

Comment:
instead, there could easily be a new method in this class with the default rendering_app. eg.:
```
def rendering_app
  'frontend'
end
```